content/en/docs/userguide/locality_loadbalance.md

```diff
@@ -0,0 +1,244 @@
+    name: http
+  selector:
+    app: helloworld
+  trafficDistribution: PreferClose
+EOF
+```
+- Start a service instance on the ambient-worker.
+```
+kubectl apply -n sample -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: helloworld-region.zone1.subzone1
+  labels:
+    app: helloworld
+    version: region.zone1.subzone1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: helloworld
+      version: region1.zone1.subzone1
+  template:
+    metadata:
+      labels:
+        app: helloworld
+        version: region1.zone1.subzone1
+    spec:
+      containers:
+      - name: helloworld
+        env:
+        - name: SERVICE_VERSION
+          value: region1.zone1.subzone1
+        image: docker.io/istio/examples-helloworld-v1
+        resources:
+          requests:
+            cpu: "100m"
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 5000
+      nodeSelector:
+        kubernetes.io/hostname: ambient-worker
+EOF
+```
+
+- Start a service instance on the ambient-worker2.
+```
+kubectl apply -n sample -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: helloworld-region.zone2.subzone2
+  labels:
+    app: helloworld
+    version: region.zone2.subzone2
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: helloworld
+      version: region1.zone2.subzone2
+  template:
+    metadata:
+      labels:
+        app: helloworld
+        version: region1.zone2.subzone2
+    spec:
+      containers:
+      - name: helloworld
+        env:
+        - name: SERVICE_VERSION
+          value: region1.zone2.subzone2
+        image: docker.io/istio/examples-helloworld-v1
+        resources:
+          requests:
+            cpu: "100m"
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 5000
+      nodeSelector:
+        kubernetes.io/hostname: ambient-worker2
+EOF
+```
+
+
+3. test on client
+- Start the test client on the ambient-worker.
+```
+kubectl apply -n sample -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sleep
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sleep
+  template:
+    metadata:
+      labels:
+        app: sleep
+    spec:
+      terminationGracePeriodSeconds: 0
+      containers:
+      - name: sleep
+        image: curlimages/curl
+        command: ["/bin/sleep", "infinity"]
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - mountPath: /etc/sleep/tls
+          name: secret-volume
+      volumes:
+      - name: secret-volume
+        secret:
+          secretName: sleep-secret
+          optional: true
+      nodeSelector:
+        kubernetes.io/hostname: ambient-worker
+EOF
+```
+
+- test
+```
+kubectl exec "$(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}')" -c sleep -- curl -sSL "http://helloworld:5000/hello"
+```
+The output is from the helloworld-region.zone1.subzone1 that is currently co-located on the ambient-worker.
```

Comment:
We need more tests, like: when a local endpoints down. It automatically failover to the remote one.